hibernate-reactive/src/main/resources/META-INF/native-image/io.micronaut.sql/hibernate-jpa-graal/resource-config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
The same as for reflect-config.json file (waiting for a stable version of hibernate-reactive)